SPECS/kata-containers-cc/mariner-coco-build-uvm.sh

```diff
@@ -15,7 +15,7 @@ export AGENT_SOURCE_BIN=${SCRIPT_DIR}/kata-agent
 pushd ${OSBUILDER_DIR}
```

Comment:
Should the spec release number get incremented too?

Reply:
going into our feature branch that prepares the new package; the number is thus already increased :)

Reply:
Ah makes sense. I missed that this was a PR into a feature branch and not the main dev branch.